inc/ticket.class.php

```diff
@@ -850,7 +850,9 @@ function defineTabs($options = []) {
       // Enable impact tab if there is a valid linked item
       foreach ($this->getLinkedItems() as $linkedItem) {
          $class = $linkedItem['itemtype'];
-         if (isset($CFG_GLPI['impact_asset_types'][$class])) {
+         if (isset($CFG_GLPI['impact_asset_types'][$class])
+            && Session::getCurrentInterface() !== "helpdesk") {
```

Comment:
```suggestion
            && Session::getCurrentInterface() === "central") {
```
maybe?

Reply:
To be consistent with "// Impact analysis should not be available outside of central "

Reply:
Done :+1: